### DISCUSSION TITLE
How can i use NAS in this docker?

### DISCUSSION THREAD

**Discussion by YellowDragonLive** [AbdBarho/stable-diffusion-webui-docker#506] [Q&A]
Hi~

I create a NAS to Stable-diffusion ,the path such as \eric\models\Stable-diffusion
but when i use this dockerfile,
I find the path changed,
like ${ROOT}\models\Stable-diffusion\

some thing happen like this
Downloading: "https://huggingface.co/runwayml/stable-diffusion-v1-5/resolve/main/v1-5-pruned-emaonly.safetensors" to /stable-diffusion-service/\eric\models\Stable-diffusion/v1-5-pruned-emaonly.safetensors

where should i change
![企业微信截图_16862816925724](https://github.com/AbdBarho/stable-diffusion-webui-docker/assets/38859059/357becf0-2aa2-4cfe-beac-92e080283f2a)
?

**Comment by DevilaN** [comment]
Why backslashes in path? Which configuration option you are using and which webui? Too many different scenarios to guess :(

**Reply by AbdBarho** [reply]
windows maybe?

**Comment by AbdBarho** [comment]
The container mounts the `data` and `outputs` folder as mentioned in the wiki
https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Usage#mounts

you can create a `docker-compose.override.yml` and change the mounts to where ever you want.

If you are running on windows, I don't think it is possible to mount across logical disks (i.e. from C:\ to D:\ or similar) although I am not really sure....

Maybe it is the easiest if you just put everything on the NAS and don't change any config? but would also be slower to load stuff
